examples/js/renderers/SVGRenderer.js

```diff
@@ -118,10 +122,26 @@ THREE.SVGRenderer = function () {
 
 	}
 
+	function getSvgColor ( color, opacity ) {
+
+		var arg = Math.floor( color.r * 255 ) + ',' + Math.floor( color.g * 255 ) + ',' + Math.floor( color.b * 255 );
+
+		if ( opacity === undefined || opacity === 1 ) return 'rgb(' + arg + ')';
+
+		return 'rgba(' + arg + ',' + opacity + ')';
+
+	}
+
+	function convert ( c ) {
+
+		return _precision < 0 ? c : c.toFixed(_precision);
```

Comment:
How about setting `_precision` to `null` by default and here check for `_precisions !== null`?